api/batch.go

```diff
@@ -109,7 +109,7 @@ func (a *API) getFullBatch(c *gin.Context) {
 	// Fetch txs forged in the batch from historyDB
 	maxTxsPerBatch := uint(2048) //nolint:gomnd
 	txs, _, err := a.h.GetTxsAPI(
-		nil, nil, nil, nil, batchNum, nil, nil, &maxTxsPerBatch, historydb.OrderAsc,
+		nil, nil, nil, nil, batchNum, nil, nil, nil, &maxTxsPerBatch, historydb.OrderAsc,
```

Comment:
This function takes many arguments now.  Please, consider now or in the future, the possibility of creating a struct with parameters to pass to the function to make it more clear which parameters are being set.  Using a struct is specially useful if some of the parameters are nil, because you can just skip setting them in the struct (and by default they will be nil)

Reply:
Opened an issue to implement this in the future: https://github.com/hermeznetwork/hermez-node/issues/692